test/e2e/Command/AssertBackwardsCompatibleTest.php

```diff
@@ -262,6 +273,21 @@ public function testWillPickLatestTaggedVersionOnNoGivenFrom(): void
         }
     }
 
+    public function testWillAllowSpecifyingBaselineConfiguration(): void
+    {
+        File\write($this->sourcesRepository . '/.roave-backward-compatibility-check.xml', self::BASELINE_CONFIGURATION);
+
+        $output = Shell\execute(__DIR__ . '/../../../bin/roave-backward-compatibility-check', [
+            '--from=' . $this->versions[0],
+            '--to=' . $this->versions[1],
+        ], $this->sourcesRepository, [], Shell\ErrorOutputBehavior::Append);
+
+        self::assertStringContainsString(
+            '.roave-backward-compatibility-check.xml" as configuration file',
+            $output,
+        );
```

Comment:
you can use `Shell\ErrorOutputBehavior::Packed` and `[$stdout, $stderr] = Shell\unpack($data)` to ensure that the string is in `stderr` not in `stdout`.

as of now, if the output goes to stdout instead of stderr, this test will still pass.